worlds/kh1/__init__.py

```diff
@@ -0,0 +1,300 @@
+import logging
+from typing import List
+
+from BaseClasses import Tutorial
+from worlds.AutoWorld import WebWorld, World
+from .Items import KH1Item, KH1ItemData, event_item_table, get_items_by_category, item_table, item_name_groups
+from .Locations import KH1Location, location_table, get_locations_by_category, location_name_groups
+from .Options import KH1Options, kh1_option_groups
+from .Regions import create_regions
+from .Rules import set_rules
+from .Presets import kh1_option_presets
+from worlds.LauncherComponents import Component, components, Type, launch_subprocess
+
+
+def launch_client():
+    from .Client import launch
+    launch_subprocess(launch, name="KH1 Client")
+
+
+components.append(Component("KH1 Client", "KH1Client", func=launch_client, component_type=Type.CLIENT))
+
+
+class KH1Web(WebWorld):
+    theme = "ocean"
+    tutorials = [Tutorial(
+            "Multiworld Setup Guide",
+            "A guide to setting up the Kingdom Hearts Randomizer software on your computer."
+            "This guide covers single-player, multiworld, and related software.",
+            "English",
+            "kh1_en.md",
+            "kh1/en",
+            ["Gicu"]
+    )]
+    option_groups = kh1_option_groups
+    options_presets = kh1_option_presets
+
+
+class KH1World(World):
+    """
+    Kingdom Hearts is an action RPG following Sora on his journey 
+    through many worlds to find Riku and Kairi.
+    """
+    game = "Kingdom Hearts"
+    options_dataclass = KH1Options
+    options: KH1Options
+    topology_present = True
+    required_client_version = (0, 3, 5)
```

Comment:
o_o

Is this copied from somewhere, or are you actually keeping a client versioning scheme? This looks suspiciously like a fairly old AP version number, so I'm just making sure